src/modules/app/route-configs/TunewsRouteConfig.js

```diff
@@ -1,21 +1,18 @@
 // @flow
 
-import type { AllPayloadsType } from './RouteConfig'
 import { RouteConfig } from './RouteConfig'
 import type { Route } from 'redux-first-router'
 import {
   createCitiesEndpoint,
   createEventsEndpoint,
   createTunewsEndpoint,
-  createTunewsLanguagesEndpoint,
-  TunewsModel,
-  Payload
+  createTunewsLanguagesEndpoint
 } from '@integreat-app/integreat-api-client'
 import fetchData from '../fetchData'
 import { cmsApiBaseUrl, tunewsApiBaseUrl } from '../constants/urls'
 
 type TunewsRouteParamsType = {| city: string, language: string |}
-type RequiredPayloadsType = {| tunews: Payload<Array<TunewsModel>> |}
+type RequiredPayloadsType = {} // Loading tunews is handled inside Page
```

Comment:
```suggestion
type RequiredPayloadsType = {||} // Loading tunews is handled inside Page
```

Reply:
Unfortunately, flow has bug there: https://github.com/facebook/flow/issues/2977

Reply:
Still...this is already 4 years old :vomiting_face: